yt_dlp/extractor/amadeustv.py

```diff
@@ -0,0 +1,60 @@
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+    ExtractorError,
+    traverse_obj,
+)
```

Comment:
Sorted and with the additional imports:
```suggestion
    ExtractorError,
    float_or_none,
    int_or_none,
    parse_iso8601,
    url_or_none,
)
from ..utils.traversal import traverse_obj
```